liquidprompt

```diff
@@ -2432,6 +2456,12 @@ _lp_default_theme_prompt() {
         LP_VCS=
     fi
 
+    if _lp_dirstack; then
+        LP_STCK=" $lp_dirstack"
+    else
+        LP_STCK=
+    fi
+
```

Comment:
Put this block in the "right order", so about https://github.com/nojhan/liquidprompt/blob/6e7f4c77044bea7ab92970e820997601926bcbea/liquidprompt#L2397